src/components/interstitials/TrendingVideos.tsx

```diff
@@ -25,7 +25,7 @@ import {
   CompactVideoPostCardPlaceholder,
 } from '#/components/VideoPostCard'
 
-const CARD_WIDTH = 100
+const CARD_WIDTH = 110
```

Comment:
Thought: I have a sense this fixes the "issue" but is it optically correct?

Reply:
Linear: https://linear.app/blueskyweb/issue/APP-1328/new-trending-video-interstitial#comment-ef404080

Reply:
Darrin and I played around with it offline and have settled on `108`